pages/about.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link type="image/png" sizes="16x16" rel="icon" href="../assets/icons8-skull-heart-16.png">
 </head>
 <body>
-  
+  <header>
+    <h1> About me! </h1>
+  </header>
+  <main>
+    <p>
+      Hi! My name is Irene Cho and I'm a full stack software engineering student 
+      at Ada Developers Academy. 
+      <br>
+      In my free time I like spending time with my cat,
+      Percy, watching YouTube videos about obscure video games I'll never play, and
+      slowly making my way through my never ending stack of books I want to read.
+    </p>
+
+    <h2 id = "percy-time">Percy (Her Royal Cuteness)</h2>
```

Comment:
HTML attributes shouldn't have whitespace, so prefer `id="percy-time"`. It's still valid HTML but it would definitely get pointed out in code review.